docs/locks.md

```diff
@@ -60,61 +37,47 @@ was successful or not. Therefore, new installations of the same easyconfig will
 * a previous installation of the same easyconfig was abruptly interrupted;
 
 
-.. _locks_removing:
-
-Removing locks
---------------
+### Removing locks
 
 If a previous installation was abruptly interrupted and a lock was left in place,
-it can be easily removed using the ``rmdir`` command (since the lock is actually an empty directory).
-
+it can be easily removed using the `rmdir` command (since the lock is actually an empty directory).
 
-.. _locks_configuration_options:
 
-Configuration options related to installation locks
----------------------------------------------------
+### Configuration options related to installation locks
 
 The behaviour of the locking mechanism in Easybuild can be tuned with the following configuration options:
 
-.. _locks_ignore:
+#### Ignoring locks (`--ignore-locks`)
 
-Ignoring locks (``--ignore-locks``)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Using the ``--ignore-locks`` configuration option, you can instruct EasyBuild to ignore any existing locks.
+Using the `--ignore-locks` configuration option, you can instruct EasyBuild to ignore any existing locks.
 Locks that exist are left untouched, even if the installation completes successfully.
 
 **Use this with caution, since installations may be (partially) overwritten if another EasyBuild session is also
 performing those installations!**
 
 
-.. _locks_wait:
-
-Waiting for locks to be removed (``--wait-on-lock``)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#### Waiting for locks to be removed (`--wait-on-lock`)
```

Comment:
```suggestion
#### Waiting for locks to be removed

*(`--wait-on-lock`)*
```
(See previous comment)

Reply:
I've modified the suggestion to add the mkdocs-autorefs syntax for the internal link